server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMailboxesMethodTest.java

```diff
@@ -269,8 +253,8 @@ public void getMailboxesShouldReturnDefaultMailboxesWhenAuthenticatedUserDoesntH
         .then()
             .statusCode(200)
             .body(NAME, equalTo("mailboxes"))
-            .body(ARGUMENTS + ".list", hasSize(4))
-            .body(ARGUMENTS + ".list.name", hasItems("INBOX", "Outbox", "Sent", "Trash"));
+            .body(ARGUMENTS + ".list", hasSize(5))
+            .body(ARGUMENTS + ".list.name", hasItems(DefaultMailboxes.INBOX, DefaultMailboxes.OUTBOX, DefaultMailboxes.SENT, DefaultMailboxes.TRASH, DefaultMailboxes.DRAFTS));
```

Comment:
maybe you could group default mailboxes into a list so that you don't have to come back to change every test the day we add one to the list ?